Dockerfile

```diff
@@ -5,5 +5,17 @@
 
 FROM alpine:3.2
 RUN apk add -U ca-certificates git openssh curl perl && rm -rf /var/cache/apk/*
+
+ENV LFS_VERSION 1.0.2
+RUN apk add --update --virtual build-dependencies curl && \
+    curl -sLO https://github.com/github/git-lfs/releases/download/v${LFS_VERSION}/git-lfs-linux-amd64-${LFS_VERSION}.tar.gz && \
```

Comment:
also wondering if `build-dependencies` is required? is that providing dynamically linked libraries?